src/lib/updating-element.ts

```diff
@@ -246,13 +245,13 @@ export abstract class UpdatingElement extends HTMLElement {
   }
 
   /**
-   * Returns true if a property should cause invalidation and update.
-   * Called when a property value is set and uses the `shouldInvalidate`
+   * Returns true if a property should request an update.
+   * Called when a property value is set and uses the `hasChanged`
    * option for the property if present or a strict identity check.
    */
-  private static _valueShouldInvalidate(value: unknown, old: unknown,
-      shouldInvalidate: ShouldInvalidate = notEqual) {
-    return shouldInvalidate(value, old);
+  private static _valueHasChanged(value: unknown, old: unknown,
```

Comment:
Isn't the point of having this static here for overriding? Why private?

Reply:
Holding off on this for now until the need is verified.